package.json

```diff
@@ -28,7 +28,6 @@
         "setup": "./scripts/setup.js",
         "build": "./scripts/build-world.sh",
         "build:lib": "./scripts/build.js",
-        "build:docs": "./scripts/build-docs.js",
```

Comment:
This change is unrelated, but this script is broken and unused so removing it here